torchrl/_utils.py

```diff
@@ -872,6 +872,58 @@ def set_mode(self, type: Any | None) -> None:
             self._mode = type
 
 
+def _standardize(input, exclude_dims: Sequence[int] = (), mean=None, std=None):
```

Comment:
There is a bit of extra compute here compared to before which every run will incur, even if `exclude_dim` is empty. I would treat that as a special case.